### ISSUE TITLE
Feature request: Add default value for `output.public`

### ISSUE THREAD

**Issue by epeli** [webpack/webpack-dev-server#1311]
* Operating System: any
* Node Version: v8.9.4
* NPM Version: 5.6.0
* webpack Version: 4.0.0-beta.1
* webpack-dev-server Version: 3.0.0-beta.2

- [ ] This is a **bug**
- [x] This is a **modification** request

### Code

```js
  // webpack.config.js
  // No config :)
```

Since #1295 got closed I'm opening a new issue of a feature request I mentioned there.

Could webpack-dev-server provide a default for the `output.publicPath` config? With this webpack-dev-server could be also used without any config like Webpack 4.

I'd propose the config to be

```
output: {
  publicPath: "/dist",
}
```

because it matches the default Webpack 4 ouput directory.

**Comment by SpaceK33z** [comment]
Yeah that might be a good idea. Are you interested in making a PR?

**Comment by epeli** [comment]
Yeah, I can come up with something for sure.

**Comment by epeli** [comment]
Looking at the code there actually seems to be a default for this:

https://github.com/webpack/webpack-dev-server/blob/v3.0.0-beta.2/bin/webpack-dev-server.js#L267

Currently it becomes `/main.js` when there is no `webpack.config.js`.

I but would still like to change it to `/dist/main.js` to match with Webpack 4.

The reasoning is that one would not have to add any config or edit the script when using the webpack-dev-server in a Webpack 4 project without the config file.

Ie.

```
webpack-dev-server
```

Would be basically same as

```
webpack --watch
python -m SimpleHTTPServer
```

but with automatic browser reloads.
